README.rst

```diff
@@ -33,6 +33,7 @@ The currently supported services include:
 * Amazon ElastiCache
 * Amazon Elastic Compute Cloud
 * Amazon Elastic MapReduce (Preview)
```

Comment:
What is the meaning of preview? Does it mean it is preview in the CLI or in preview for the service in general.